packages/docs/actions/SetFocus.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020-2024 Lowdefy, Inc
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/actions.yaml.njk
+  vars:
+    pageId: SetFocus
+    pageTitle: SetFocus
+    filePath: actions/SetFocus.yaml
+    types: |
+      ```
+      (params: string): void
+      ```
+    description: |
+      The `SetFocus` action sets focus on the specified HTML element, provided it can receive focus. It takes an id of the HTML element as a parameter.
+      By default, when focusing an element, the browser will scroll it into view and may provide visible indication of the focus, typically by displaying a "focus ring" around the element.
+    params: |
```

Comment:
Please add:

Note that for blocks like `TextInput`, the id of the HTML input element is not equal to the block id, since the block includes other html elements like the label, extra text and icons. You can inspect the HTML elements in the browser to find the correct element id.

Reply:
0d11ab656b390b068df51a96141ed45f23eef6a9